docs/Configmap_Properties.md

```diff
@@ -8,12 +8,50 @@ This document explains configmap variables.
 | iam.managed.permission.boundary.policy| User managed permission boundary policy|k8s-iam-manager-cluster-permission-boundary       |Required            |
-| webhook.enabled                   |  Enable webhook?              | false              | Required           |
-| iam.role.max.limit.per.namespace      | Maximum number of roles per namespace |   1        | Required |
-| aws.region                        | AWS Region                    | us-west-2          | Required |
-| iam.default.trust.policy| Default trust policy role. This must follow v1alpha1.AssumeRolePolicyDocument syntax|           | Optional |
+| webhook.enabled                   |  Enable webhook?              | `false             | Required           |
+| iam.role.max.limit.per.namespace  | Maximum number of roles per namespace |   1        | Required |
+| aws.region                        | AWS Region                    | `us-west-2`        | Required |
+| iam.default.trust.policy          | Default trust policy role. This must follow v1alpha1.AssumeRolePolicyDocument syntax|           | Optional |
+| [iam.role.prefix](#iamroleprefix) | See docs below...           | `k8s`              | Optional           |
+| [iam.role.separator](#iamroleseparator) | See docs below...     | `-`                | Optional           |
 | iam.role.derive.from.namespace    | Derive iam role name from namespace? if true it will be k8s-<namespace> | false | Optional|
 | controller.desired.frequency      | Controller frequency to check the state of the world (in seconds) | 300  | Optional |
 | k8s.cluster.name                  | Name of the cluster           |                    | Optional | 
 | k8s.cluster.oidc.issuer.url       | OIDC issuer of the cluster    |                    | Optional |
-| iam.irsa.enabled                  | Enable IRSA option?           | false              | Optional |
\ No newline at end of file
+| iam.irsa.enabled                  | Enable IRSA option?           | `false`            | Optional |
+
+
+## `iam.role.prefix`
+
+_Default_: `k8s`
+
+All IAM roles created and managed by the controller will use this prefix. This
+helps organize IAM roles within your AWS Account, and can be used to ensure
+uniqueness between different EKS Clusters within the same AWS account.
+
+**Critical Note: Read [this](#note-about-changing-iamroleprefix-and-iamroleseparator)
+before changing this setting**
+
+**Note: Changes to your IAM Policy may be required if you customize this**
+
+## `iam.role.separator`
+
+_Default_: `-`
+
+This is the separator between the [IAM Role Prefix](#iamroleprefix) and the
+desired IAM Role Name.
+
+**Critical Note: Read [this](#note-about-changing-iamroleprefix-and-iamroleseparator)
+before changing this setting**
+
+**Note: Changes to your IAM Policy may be required if you customize this**
+
+## Note about changing `iam.role.prefix` and `iam.role.separator`
```

Comment:
Great!! This is one of the main reason I was little hesitant to do this in first place and created an issue https://github.com/keikoproj/iam-manager/issues/47 where if someone wants to override that can be done through CR itself.

Reply:
Heh sure enough, I didn't see #47 - my issue #62 is clearly a dupe. I am planning on adding in the ability for us to templatize the role name as well. A bit of Go templating and we should be able to do something like: `iam.role.name: {{ metadata.namespace}}-{{metadata.name}}` for example.